Marlin/src/lcd/extensible_ui/ui_api.cpp

```diff
@@ -445,7 +495,8 @@ namespace UI {
   }
 
   uint32_t getProgress_seconds_elapsed() {
-    return IFPC(print_job_timer.duration() / 1000UL, 0);
+    const duration_t elapsed = print_job_timer.duration();
+    return elapsed.value;
```

Comment:
I missed that the `duration()` method in `stopwatch.h` divides the returned value by 1000 before returning it. Are we guaranteed to have `print_job_timer` defined, if `printcounter.h` isn't included? Let's add a Travis CI test for `EXTENSIBLE_UI` with and without all the extras so we can confirm everything compiles ok.

Reply:
Perhaps add the header anyway?